mlir-sycl/lib/Dialect/SYCL/Utils/Utils.cpp

```diff
@@ -12,5 +12,42 @@
 
 #include "mlir/Dialect/SYCL/Utils/Utils.h"
 
-using namespace mlir;
-using namespace mlir::sycl;
+namespace mlir {
```

Comment:
NIT: Use namespace qualifiers instead of blocks ([see](https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions)).

Reply:
done.